python/paddle/v2/fluid/evaluator.py

```diff
@@ -165,28 +156,23 @@ def __init__(self,
             label=label,
             chunk_scheme=chunk_scheme,
             num_chunk_types=num_chunk_types,
-            excluded_chunk_types=excluded_chunk_types,
-            **kwargs)
+            excluded_chunk_types=excluded_chunk_types, )
```

Comment:
The line 153:
```cpp
kwargs = {'main_program': main_program}
```
should also be removed. I can not commit on unchanged line so I leave it here.